src/site/_includes/components/header/header.html

```diff
@@ -0,0 +1,176 @@
+                fill="#231F20"
+                fill-rule="nonzero"
+              >
+                <g id="g51" transform="translate(1.387097, 0.000000)">
+                  <path
+                    d="M20.9229677,10.2536 C26.6114516,10.2536 31.9281935,11.7992 36.5028387,14.497 L41.3881935,6.0662 C35.3820645,2.5102 28.3897097,0.483 20.9229677,0.483 C13.4631613,0.483 6.47912903,2.5102 0.471612903,6.0662 L5.35558065,14.497 C9.933,11.7992 15.2566774,10.2536 20.9229677,10.2536 Z"
+                    id="path53"
+                  ></path>
+                </g>
+                <g id="g55" transform="translate(0.000000, 12.600000)">
+                  <path
+                    d="M26.8264516,2.2918 C24.8429032,2.2708 18.8256774,2.1532 11.0898387,5.7624 C10.1036129,6.2244 9.40590323,6.6094 9.23806452,6.3644 C8.90516129,5.831 11.3644839,3.7226 14.0027419,2.1812 C14.9570645,1.6086 15.5021935,1.1088 15.2885806,0.6832 C14.9709355,0.063 13.0803226,-0.0644 7.80519355,2.66 C5.10312903,4.1818 2.65906452,6.1278 0.567322581,8.393 L1.42732258,9.2134 C2.27483871,8.3076 3.18893548,7.469 4.12106452,6.6472 C4.74387097,6.1026 7.62764516,3.927 7.88009677,4.5836 C7.97719355,4.8468 7.79270968,5.2304 6.77180645,6.6388 C5.88129032,7.8848 5.29593548,9.0496 5.7176129,9.4052 C6.13790323,9.765 6.64558065,9.5606 7.84819355,8.8914 C8.74009677,8.3944 11.5697742,6.881 16.0306774,5.5272 C20.4915806,4.1832 21.5762903,4.6942 21.6844839,5.0232 C21.8204194,5.4656 19.766129,6.1992 18.5551935,6.8054 C17.5412258,7.329 13.9625161,9.4514 12.0205806,11.0306 C12.0205806,11.0306 13.902871,14.2912 13.9111935,14.2912 C15.9141613,13.1012 23.5459677,11.1524 24.3615806,12.5608 C25.2562581,14.1064 17.1348065,19.8674 17.1348065,19.8674 L22.3100645,28.8442 C22.3100645,28.8442 34.085129,15.1942 32.2083871,12.1996 C30.315,9.2204 23.2269355,9.772 20.4305484,10.0394 C19.6620968,10.108 18.7604839,10.2774 18.5704516,9.8938 C18.3596129,9.4696 18.6023548,7.7644 28.0526452,5.5048 C30.1540968,5.0078 30.9724839,4.7474 30.8573548,3.8094 C30.7602581,3.0044 28.8113871,2.3058 26.8264516,2.2918 Z"
+                    id="path57"
+                  ></path>
+                </g>
+              </g>
+              <text
+                id="CMD"
+                font-family="OpenSans-Bold, Open Sans"
+                font-size="36"
+                font-weight="bold"
+                fill="#000000"
+              >
+                <tspan x="-2" y="38">CMD</tspan>
+              </text>
+              <line
+                x1="109.5"
+                y1="4.5"
+                x2="109.5"
+                y2="45.5"
+                id="Line"
+                stroke="#231F20"
+                stroke-linecap="square"
+              ></line>
+            </g>
+          </svg>
+        </a>
+      </div>
+      {{ button({ type: 'button', text: 'Menu' }) }}
+    </div>
+  </div>
+  <nav>
+    {% for item in site.navigation.mainMenu.content.navigation_link %}
```

Comment:
🤢 Hier moeten we later echt iets op verzinnen dat het niet zo diep genest moet zijn.

Reply:
Voor nu is dit prima by the way